xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-xwiki/src/test/java/org/xwiki/rendering/internal/resolver/DefaultResourceReferenceEntityReferenceResolverTest.java

```diff
@@ -387,4 +389,42 @@ public void resolveTypeAttachment() throws ComponentLookupException
             new AttachmentReference(ATTACHMENT, new DocumentReference(CURRENT_WIKI, CURRENT_SPACE, CURRENT_PAGE)),
             this.mocker.getComponentUnderTest().resolve(attachmentResource(ATTACHMENT, true), null));
     }
+
+    class VoidResourceReferenceEntityReferenceResolve extends AbstractResourceReferenceEntityReferenceResolver
+    {
+        VoidResourceReferenceEntityReferenceResolve()
+        {
+            this.documentAccessBridge = DefaultResourceReferenceEntityReferenceResolverTest.this.bridge;
+        }
+
+        @Override
+        protected EntityReference resolveTyped(ResourceReference resourceReference, EntityReference baseReference)
+        {
+            return null;
+        }
+    }
+
+    @Test
+    public void trySpaceSiblingFallback() {
```

Comment:
code style :)

Reply:
Rha. Just for the record: the checkstyle is not launched against test, right? Bc the build was passing even with quality profile.

Reply:
Yes seems it's not since the default in the checkstyle plugin is off, see https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html#includeTestSourceDirectory

Reply:
Seems auto applying checkstyle on save is not supported by IDEA: https://stackoverflow.com/questions/8184335/how-to-set-up-checkstyle-idea-plugin-to-automatically-enforce-a-rule-upon-saving

Reply:
Maybe with a git hook?

Reply:
https://gist.github.com/davetron5000/37350 I'll try that one.